dashboards-observability/public/components/custom_panels/home.tsx

```diff
@@ -49,8 +49,8 @@ export const Home = ({ http, chrome, parentBreadcrumb, pplService, renderProps }
   const [toasts, setToasts] = useState<Array<Toast>>([]);
   const [loading, setLoading] = useState(false);
   const [toastRightSide, setToastRightSide] = useState<boolean>(true);
-  const [start, setStart] = useState<ShortDate>('now-30m');
-  const [end, setEnd] = useState<ShortDate>('now');
+  const [start, setStart] = useState<ShortDate>('');
+  const [end, setEnd] = useState<ShortDate>('');
```

Comment:
will there be a default time range?

Reply:
it is always fetched from the backend. Whenever user create a new one default time is set to 1day in panels adaptor.